src/test/java/org/fundacionjala/coding/jimmy/Kata2Test.java

```diff
@@ -0,0 +1,59 @@
+package org.fundacionjala.coding.jimmy;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test class created by Jimmy Romero.
+ */
+public class Kata2Test {
+
+    private Kata2 kata2;
+
+    /**
+     * It initializes kata2.
+     */
+    @Before
+    public void initialize() {
+        kata2 = new Kata2();
+    }
+
+
+    /**
+     * This test is performed in order to compare the expected and actual result.
+     */
+    @Test
+    public void test1() {
+        String stringWords = "sort the inner content in descending order";
+        assertEquals("srot the inner ctonnet in dsnnieedcg oredr", kata2.sortInnerContent(stringWords));
+    }
+
+    /**
+     * This test is performed in order to compare the expected and actual result.
+     */
+    @Test
+    public void test2() {
+        String stringWords = "wait for me";
+        assertEquals("wiat for me", kata2.sortInnerContent(stringWords));
+    }
+
+    /**
+     * This test is performed in order to compare the expected and actual result.
+     */
+    @Test
+    public void test3() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)